src/main/webapp/step2-startup.js

```diff
@@ -9,10 +11,56 @@ function startup() {
     let container = document.getElementsByClassName('main-interface')[0];
     const renderer = new Step2Renderer(container);
 
-    const eventManager = new EventManager();
     renderer.render(eventManager.getEvents());
 }
 
+/**
+ * Checks the value in all radio boxes and add data to LocalStorage
+ * TODO (remusn@) Check if radio boxes are empty
+ */
+function checkRadioBoxes() {
+    const size = document.getElementsByClassName('info-pane').length;
+    for (let index = 0; index < size; ++index)  {
+        const checkbox = document.getElementsByClassName('info-pane')[index];
+        addTransportationToEvent(checkbox, index)
+    }
+}
+
+/**
+ * Update events from LocalStorage
+ * For a transportation mode with index n (from event n to event n+1)
+ * the information is added to the n-th event data
+ */
+function addTransportationToEvent(checkbox, index) {
+    const transportation = [ "Walk", "Public transportation", "Car", "Bike" ];
+    
+    // Two consecutive elements are in the same location
+    if (!checkbox.getElementsByTagName('input').length) {
+        return;
+    }
+
+    for (let step = 0; step < 4; ++step) {
+        if (checkbox.getElementsByTagName('input')[step].checked) {
+            eventManager.updateEvent(index, 
+                { travelMode: transportation[step] } 
+            )
+            break; 
+        }
+    }
+}
+
+/**
+ * Create the window.itnerary object only once
+ */
+if (window.itnerary == null) {
+    window.itnerary = {};
+}
+
+/**
+ * Adds the methods of transportation to LocalStorage
+ */
+window.itnerary.addTransportToLocalStorage = checkRadioBoxes;
```

Comment:
I believe - you forgot to rename the right side.

Reply:
Do you mean checkRadioBoxes?

Reply:
I thought it had a good name since the functions verifies if there is value in the radio boxes (atm it doesn't but in the next PR it will) and after that adds data to LocalStorage

Reply:
I am sorry, I didn't understood well the first time but now I got it

Reply:
Ah, you have two functions here... I thought you renamed the checkRadioBoxes into addTransportationToEvent.

checkRadioBoxes doesn't describe the purpose of this method.